lib/iris/tests/experimental/test_animate.py

```diff
@@ -40,6 +40,7 @@
 @tests.skip_plot
 class IntegrationTest(tests.GraphicsTest):
     def setUp(self):
+        super(IntegrationTest, self).setUp()
```

Comment:
@dkillick Thanks. Yes, we do indeed need to follow a pattern here ...

A `GraphicsTest` subclass **must** call the `GraphicsTest.setUp` method (in order to `acquire` the non-reentrant lock) **if** the subclass in question overrides the inherited `setUp` method. If the subclass doesn't specialise the `setUp` method then, thanks to inheritance, the right thing does happen, in that the `unittest` calls the (inherited) `GraphicsTest.setUp` for the subclass (this also applies to `GraphicsTest.tearDown` btw to `release` the lock).

In **all** of the test cases that you identified above, there is **no need** to explicitly call `GraphicsTest.setUp`, simply because the subclass **does not** specialise the `setUp` method.

Note that, you incorrectly mentioned `test_mapping.TestBasic`:
```python
class TestBasic(tests.GraphicsTest):
    def setUp(self):
        super(TestBasic, self).setUp()
        self.cube = iris.tests.stock.realistic_4d()
```
which does apply the pattern, and so does `test_plot.Test1dPlotMultiArgs`:
```python
class Test1dPlotMultiArgs(tests.GraphicsTest):
    # tests for iris.plot using multi-argument calling convention

    def setUp(self):
        super(Test1dPlotMultiArgs, self).setUp()
        self.cube1d = _load_4d_testcube()[0, :, 0, 0]
        self.draw_method = iplt.plot
```
So, I'd argue that this isn't an issue, unless I'm missing something ...